http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200501_kubernetes_tables.xml

```diff
@@ -0,0 +1,113 @@
+                <constraints nullable="false"/>
+            </column>
+            <column name="creator" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="createdDate" type="TIMESTAMP(6)" defaultValue="0000-00-00 00:00:00.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="destroyedDate" type="TIMESTAMP(6)" defaultValue="1970-01-01 00:00:01.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dateAccessed" type="TIMESTAMP(6)" defaultValue="0000-00-00 00:00:00.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="machineType" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="numNodes" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="autoScalingEnabled" type="boolean">
+                <constraints nullable="false"/>
+            </column>
+            <column name="autoScalingMin" type="INT"/>
+            <column name="autoScalingMax" type="INT"/>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="clusterId" baseTableName="NODEPOOL" constraintName="FK_NODEPOOL_CLUSTER_ID" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="KUBERNETES_CLUSTER"/>
+        <createIndex indexName="IDX_NODEPOOL_ID" tableName="NODEPOOL">
+            <column name="id"/>
+        </createIndex>
+        <addUniqueConstraint columnNames="clusterId, nodepoolName" constraintName="IDX_NODEPOOL_UNIQUE" tableName="NODEPOOL"/>
+    </changeSet>
+
+    <changeSet logicalFilePath="leonardo"  id="namespace_table" author="jcanas">
+        <createTable tableName="NAMESPACE">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="clusterId" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="namespaceName" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="clusterId" baseTableName="NAMESPACE" constraintName="FK_NAMESPACE_CLUSTER_ID" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="KUBERNETES_CLUSTER"/>
+        <createIndex indexName="IDX_NAMESPACE_ID" tableName="NAMESPACE">
+            <column name="id"/>
+        </createIndex>
+        <addUniqueConstraint columnNames="clusterId, namespaceName" constraintName="IDX_NAMESPACE_UNIQUE" tableName="NAMESPACE"/>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
Migration seems fine, but as a sanity check I like to do:
```
DESCRIBE <tableName>
SHOW INDEX FROM <tableName>
```
on any new/changed tables. Can do this on the local unit-test MySQL DB.